From f35cdc5e7bc15cd457eaf6cfb10f4a677f9ea33a Mon Sep 17 00:00:00 2001
From: GwynBleidD <gbdlin@users.noreply.github.com>
Date: Thu, 27 Jun 2024 22:52:57 +0200
Subject: [PATCH] hwdb: Added StarLabs StarLite position sensor mapping

Default mapping for position sensor in StarLabs StarLite does
not match screen orientation. Added fixed mapping matrix.

(cherry picked from commit 1bd5db86f50feb2efd8e54e8156816d9727f6f2f)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index b45db6292b..cc841681d7 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -989,6 +989,13 @@ sensor:modalias:acpi:KIOX000A*:dmi:*:svnRCA:pnW101SA23T1:*
 sensor:modalias:acpi:BOSC0200*:dmi:bvnAmericanMegatrendsInc.:bvrSCH12i.WJ210Z.KtBJRCA03:*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
 
+#########################################
+# StarLabs Systems
+#########################################
+# StarLite
+sensor:modalias:acpi:KIOX000A*:dmi:*:svnStarLabs:pnStarLite:*
+ ACCEL_MOUNT_MATRIX=1, 0, 0; 0, -1, 0; 0, 0, 1
+
 #########################################
 # Teclast
 #########################################
